tools/bazel_java_integration_test.bzl

```diff
@@ -36,6 +38,9 @@ bazel_java_integration_test_properties_ = rule(
     attrs = {
         "bazel_version": attr.string(mandatory = True),
         "external_deps": attr.label_list(allow_files = True),
+        "_java_runtime": attr.label(
```

Comment:
You can rename `_java_runtime` to `_test_java_runtime` or something similar to distinguish between the test external javabase and the internal one...

Reply:
+1

Reply:
Thanks! Not sure that would be correct. I agree that’s what I’m using it *for* but
OTOH I’m not using a custom java runtime for the test but rather passing
the real java runtime into the test.